commands/git-msg/action.yml

```diff
@@ -7,7 +7,7 @@ outputs:
 runs:
   using: "composite"
   steps:
-    - run: echo "::set-output name=msg::`git log -1 --pretty=%B | head -n 1`"
+    - run: echo "msg=`git log -1 --pretty=%B | head -n 1`" >> $GITHUB_OUTPUT
```

Comment:
I recently learned that if the right part contains e.g "space character" - it'll break.
Solution - double-quote the right part.

Reply:
@kirillgroshkov I'm not sure I follow here...

Do you mean the code inside the backticks?

Reply:
I mean `msg=Hello world` would "fail" because of the space, while `msg="Hello world"` would work as expected

Reply:
👍 🥂

![Screenshot 2023-07-07 at 08 12 17](https://github.com/NaturalCycles/actions/assets/58119759/d2c44ec2-ec2f-421d-9ce1-77256b0c6d83)

Reply:
In this case though, everything is wrapped as a string (we redirect the string "msg=bla bla bla" into GITHUB_ENV) So the second bla will not be read as a if it were a separate command.

Also, this exact method is used in existing code below on line 13 since may this year, so it doesn't throw an error.

Reply:
![Screenshot 2023-07-07 at 08 20 47](https://github.com/NaturalCycles/actions/assets/58119759/7fd345e6-6668-4ab3-a6d5-4d2f31fc2baf)

gives

![Screenshot 2023-07-07 at 08 21 07](https://github.com/NaturalCycles/actions/assets/58119759/dcce875c-6730-4794-9251-5e95258f544d)